README.md

```diff
@@ -13,16 +13,24 @@ after a set time period discourages users from using the server as a permanent f
 * yarn (https://yarnpkg.com/ - for the kiwiirc plugin UI)
 * dep (https://github.com/golang/dep - for the server)
 
+
+#### Downloading the file upload server
+
+```console
+$ go get github.com/kiwiirc/plugin-fileuploader
+$ cd ~/go/src/github.com/kiwiirc/plugin-fileuploader
```

Comment:
Let's make this work for any (or no) `$GOPATH`.

I think this should do the trick:

```console
$ cd "$(go env GOPATH)/src/github.com/kiwiirc/plugin-fileuploader"
```